test/parallel/test-fs-watchfile.js

```diff
@@ -67,7 +67,7 @@ fs.watchFile(enoentFile, {interval: 0}, common.mustCall(function(curr, prev) {
 // Watch events should callback with a filename on supported systems.
 // Omitting AIX. It works but not reliably.
 if (common.isLinux || common.isOSX || common.isWindows) {
-  const dir = common.tmpDir + '/watch';
+  const dir = `${common.tmpDir}/watch`;
```

Comment:
This is Ok.
But it would be much better to use `path.join` instead, see https://github.com/nodejs/node/pull/14272#discussion_r127608804